javaee/ejb-mdb-activemq/src/main/resources/project-defaults.yml

```diff
@@ -0,0 +1,28 @@
+thorntail:
+  deployment:
+    org.apache.activemq:activemq-rar.rar:
+  resource-adapters:
+    resource-adapters:
+      activemq-ra:
+        archive: activemq-rar.rar
+        transaction-support: XATransaction
+        config-properties:
+          UseInboundSession:
+            value: false
+          UserName:
+            value: amq
+          Password:
+            value: amq
+          ServerUrl:
+            value: tcp://localhost:61616?jms.rmIdFromConnectionId=true
+        connection-definitions:
+          default:
+            jndi-name: java:global/jms/activemq
+            class-name: org.apache.activemq.ra.ActiveMQManagedConnectionFactory
+            same-rm-override: false
+  ejb3:
+    default-resource-adapter-name: activemq-ra
+  ee:
+    annotation-property-replacement: true
```

Comment:
I think this isn't needed in this module, right?

Reply:
Probably, yea. I think I'd rather keep it, because it's most likely required in any kind of actual production code.

Reply:
Ok thanks